lib/logflare/backends/adaptor/postgres/pipeline.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Logflare.Backends.Adaptor.Postgres.Pipeline do
+  @moduledoc false
+  use Broadway
+
+  alias Broadway.Message
+  alias Logflare.Buffers.BufferProducer
+  alias Logflare.Backends.Adaptor.Postgres.LogEvent
+
+  @spec start_link(PostgresAdaptor.t()) :: {:ok, pid()}
+  def start_link(adaptor_state) do
+    Broadway.start_link(__MODULE__,
+      name: adaptor_state.pipeline_name,
+      producer: [
+        module:
+          {BufferProducer,
+           buffer_module: adaptor_state.buffer_module, buffer_pid: adaptor_state.buffer_pid},
+        transformer: {__MODULE__, :transform, []},
+        concurrency: 1
+      ],
+      processors: [
+        default: [concurrency: 3, min_demand: 1]
+      ],
+      context: adaptor_state
+    )
+  end
+
+  # see the implementation for Backends.via_source_backend/2 for how tuples are used to identify child processes
+  def process_name({:via, module, {registry, identifier}}, base_name) do
+    new_identifier = Tuple.append(identifier, base_name)
+    {:via, module, {registry, new_identifier}}
+  end
+
+  def handle_message(_processor_name, message, adaptor_state) do
+    Message.update_data(message, &process_data(&1, adaptor_state))
+  end
+
+  defp process_data(log_event, %{repository_module: repository_module}) do
+    timestamp =
+      log_event.body["timestamp"]
+      |> DateTime.from_unix!(:microsecond)
+      |> DateTime.to_naive()
+
+    params = %{
```

Comment:
should ingest the entire body as is. understand if you want to separate the schema/dynamic migration changes for now, but this would be a regression as it won't ingest other top level keys.

Reply:
wanted to make something similar to what we have on `lib/logflare/source/bigquery/pipeline.ex` but also wanted to avoid to have a single :map type for the full log since that would make the endpoint construction way more difficult for us (we would need a lot of jsonb queries 😅)

I can make the metadata to receive everything (aka the body)

Reply:
Did the change to ingest the full body